### PR TITLE
[SERV-778] Make page title vary by page

### PR DESCRIPTION
**Pull request by markmatney**
- Make page title depend on current page
- Add note about dependency installation

### CODE REVIEW

File: ucla-prrla/src/app/detail-item/detail-item.component.ts [hunk 1]
```diff
@@ -110,6 +112,13 @@ export class DetailItemComponent implements OnInit {
 
         this.testService.getItemById(this.id).subscribe(data => {
             this.item = data;
+
+            if (data instanceof Object) {
+                this.title.setTitle(`${data.title} | PRL`);
+            } else {
+                this.title.setTitle("FUBAR | PRL");
```
Comment (DRickard) [inline ucla-prrla/src/app/detail-item/detail-item.component.ts]: Cute. But at the risk of sounding like a bluenose, maybe use a less... incendiary term? I suspect most people aren't as clueless as Upham in _Saving Private Ryan_.

Comment (markmatney) [reply ucla-prrla/src/app/detail-item/detail-item.component.ts]: hmm, my search pattern for swear words needs updating to account for abbreviations. Anyway, changed

**Review by ksclarke — APPROVED** [review]

**Review by DRickard — APPROVED** [review]

File: ucla-prrla/src/app/detail-item/detail-item.component.ts (hunk 1 above)
Comment (ksclarke) [reply ucla-prrla/src/app/detail-item/detail-item.component.ts]: @markmatney out of curiosity, do you have a git hook, or something, for detecting swear words before push or was that just a throwaway comment?

Comment (markmatney) [reply ucla-prrla/src/app/detail-item/detail-item.component.ts]: No, just kidding. LOL. Although that's a good idea

**Pull request merged**